src/services/DBService/project.ts

```diff
@@ -112,10 +125,22 @@ class ProjectService {
         throw Error("Update data type is not valid");
       }
 
-      const updatedProject = await Project.updateOne(
-        { ...targetOptions },
-        { $set: { ...updateOptions } },
-      );
+      let updatedProject;
+
+      if (updateOptions.deployments) {
+        const [deploymentId] = updateOptions.deployments;
+
+        updatedProject = await Project.findOneAndUpdate(
+          { ...targetOptions },
+          { $set: { ...updateOptions } },
+          { $push: { deployments: deploymentId } },
+        );
+      } else {
+        updatedProject = await Project.findOneAndUpdate(
+          { ...targetOptions },
+          { $set: { ...updateOptions } },
+        );
+      }
```

Comment:
```suggestion
      const updatedProject = !!updateOptions.deployments
        ? await Project.findOneAndUpdate(
            { ...targetOptions },
            { $set: { ...updateOptions } },
            { $push: { deployments: updateOptions.deployments[0] } },
          )
        :  await Project.findOneAndUpdate(
             { ...targetOptions },
             { $set: { ...updateOptions } },
            );
```

삼항연산자를 활용하는게 조금 더 명확하고 예측하기 쉬워지는 것 같아요~

Reply:
그렇네요. updatedProject에는 어떤 경우에도 할당이 되니까 삼항연산자가 더 좋은 선택같네요! 변경하겠습니다.